3rdparty/kleidicv/CMakeLists.txt

```diff
@@ -20,4 +20,5 @@ else()
   set(THE_ROOT "${OpenCV_BINARY_DIR}/3rdparty/kleidicv/kleidicv-${KLEIDICV_SRC_COMMIT}")
 endif()
 
+option(KLEIDICV_ENABLE_SME2 "" OFF) # not compatible with some CLang versions in NDK
```

Comment:
`if(ANDRROID)` ?

Reply:
It's not specific to Android, it's clang side effect.